src/main/java/io/github/bbortt/event/planner/repository/SectionRepository.java

```diff
@@ -17,7 +17,7 @@
     List<Section> findAllByLocationId(Long locationId, Sort sort);
 
     @Query(value = "SELECT s FROM Section s" + "  LEFT JOIN FETCH s.events" + "  WHERE s.location.id = :locationId" + "    ORDER BY s.name")
-    List<Section> findAllByLocationIdInclusiveEvents(@Param("locationId") Long locationId);
+    List<Section> findAllByLocationIdJoinEventsOrderByNameAsc(@Param("locationId") Long locationId);
```

Comment:
Der Name hast du angepasst, damit JPA das richtige Query ausführt?

Reply:
nein, wollte das eigentlich im andern PR noch drinne haben aber falsch commited. das query ist in `@Query` definiert, verwende hier ein JPA query.